language/move-binary-format/src/file_format_common.rs

```diff
@@ -379,8 +383,12 @@ pub const VERSION_4: u32 = 4;
 ///  +/- script and public(script) verification is now adapter specific
 pub const VERSION_5: u32 = 5;
 
+/// Version 6: changes compared with version 5
+///  + metadata
+pub const VERSION_6: u32 = 6;
+
```

Comment:
I opened a discussion about this on Discord underneath this PR, please see there. Its unclear to me what our current release model is. https://discord.com/channels/964205366541963294/974090756052647957/974537997779357706

Reply:
After discussion on Discord, we concluded we don't need v6 as of yet. Instead we are folding this change into v5. The downwards serialization logic is still there, but not complete to generate v4, mostly because I do not know how to undo the v5 changes before metadata. See also #145.